src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/PhysicalProjectTreeStorageTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.ProjectSystem.Utilities;
+using Xunit;
+
+namespace Microsoft.VisualStudio.ProjectSystem
+{
+    [ProjectSystemTrait]
+    public class PhysicalProjectTreeStorageTests
+    {
+        [Fact]
+        public void CreateFolderAsync_NullAsPath_ThrowsArgumentNull()
+        {
+            var storage = CreateInstance();
+
+            Assert.Throws<ArgumentNullException>("path", () => {
+
```

Comment:
Ha, interestingly I do that deliberately everywhere I write a lambda statement, because I put the brace on the previous line. 

We should write up the code guidelines and then apply this universally on whatever we decide rather than doing this piecemeal.